attest/verifier/types/src/verification.rs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
```

Comment:
Same review as other PR. Everyone seems to be moving towards workspace/types pattern - let's discuss if it's something we want to make canonical!